CHANGELOG.md

```diff
@@ -1,3 +1,89 @@
+ * Upgrade to Pulpcore 3.28 ([#36637](https://projects.theforeman.org/issues/36637), [3e134772](https://github.com/Katello/katello.git/commit/3e134772909d6dc6f8002a86ab47e6c456c46022))
+
+### Content Views
+ * Better content view filtering representation for packages vs modular packages ([#36683](https://projects.theforeman.org/issues/36683), [7bf5ec6b](https://github.com/Katello/katello.git/commit/7bf5ec6b164600eca14f8ce10d895fae70590100))
+ * "Add content view" window and "Update version" window should display content view version, description and publishing date ([#36648](https://projects.theforeman.org/issues/36648), [c9fe25d6](https://github.com/Katello/katello.git/commit/c9fe25d6705d5cf84dc60fefb521e8d3e939688b))
+
+### API
+ * Remove katello-agent API endpoints ([#36665](https://projects.theforeman.org/issues/36665), [bb738c6a](https://github.com/Katello/katello.git/commit/bb738c6ae3c6336706606142b856496b8d3f6ab1), [ff7d5d71](https://github.com/Katello/hammer-cli-katello.git/commit/ff7d5d71b55920284974134c11d4611c40675bbc))
+
+### Hosts
+ * New host details UI - Remove katello-agent code ([#36664](https://projects.theforeman.org/issues/36664), [acd85a45](https://github.com/Katello/katello.git/commit/acd85a45b04061c4f3499e2fe1076cb0420008ed))
+
+### Alternate Content Sources
+ * Add option --all to "hammer alternate-content-source refresh" command to refresh all ACS's ([#36589](https://projects.theforeman.org/issues/36589), [9599a2c5](https://github.com/Katello/katello.git/commit/9599a2c58d9dd804d463c985b9bcfbdfbd02fd1b), [1f29a1a8](https://github.com/Katello/hammer-cli-katello.git/commit/1f29a1a8acfcdb5689ee773da06327c061356bb0))
+
+### Activation Key
+ * Add activation key details top bar ([#36576](https://projects.theforeman.org/issues/36576), [b4f50250](https://github.com/Katello/katello.git/commit/b4f50250492191c8caf610593f564276b8c2a098))
+
+## Bug Fixes
+
+### Web UI
+ * about page broken after katello-agent removal ([#36722](https://projects.theforeman.org/issues/36722), [5d2f049c](https://github.com/Katello/katello.git/commit/5d2f049c72b900f45de7e062edc162207b6743ba))
+ * Legacy Content Host UI - Remove katello-agent code ([#36649](https://projects.theforeman.org/issues/36649), [5dc6942e](https://github.com/Katello/katello.git/commit/5dc6942e738d55c0a4a672cab881245aa482f3ed))
+ * Content tab subtabs disappear from host details page when you click them ([#36613](https://projects.theforeman.org/issues/36613), [d12f0b13](https://github.com/Katello/katello.git/commit/d12f0b1303608401912e2e28e3f1f2ce1f53ae5e))
+
+### Hosts
+ * Discovery Provisioning fails as the sync media is getting diminished from the host page even though its synced. ([#36721](https://projects.theforeman.org/issues/36721), [58dd1b73](https://github.com/Katello/katello.git/commit/58dd1b73e79ce27895107adbfaa5d7122a95204b))
```

Comment:
I think some of these changelog entries are duplicates: https://github.com/Katello/katello/pull/10759/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR119

Reply:
Will pull down the changelog and fix it up and repush, not sure why it did that. Good catch

Reply:
nice catch thanks! @ianballou